### DISCUSSION TITLE
Welcome to AES-Encrypted-Password-Manager Discussions!

### DISCUSSION THREAD

**Discussion by akvsx** [Akshay-Vs/PassLock#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jim-Huo** [comment]
Hai

**Comment by Codder-Girl** [comment]
Wow

**Comment by Codder-Girl** [comment]
Nice

**Comment by Erric-Muller** [comment]
Awesome

**Comment by Erric-Muller** [comment]
😍

**Comment by codey-ash** [comment]
👍👍
